terraform-aws-github-runner/modules/runners/lambdas/runners/src/scale-runners/scale-up.ts

```diff
@@ -248,20 +291,41 @@ async function allRunnersBusy(
     return 0;
   }
 
-  // Have a fail safe just in case we're likely to need more runners
+  if (requestedCount > maxAllowedScaleUp) {
+    console.info(
+      `Requested count ${requestedCount} is higher than max allowed scale up ${maxAllowedScaleUp}, ` +
+        `will scale up ${maxAllowedScaleUp} instead`,
+    );
+    requestedCount = maxAllowedScaleUp;
```

Comment:
It is not nice to overwrite a function parameter, it is better to create a `let toIncreaseCount` (or something, I am bad at naming) for that